src/locale/locales/es/messages.po

```diff
@@ -1338,47 +1338,47 @@ msgstr "Borrar consulta de búsqueda"
 
 #: src/view/screens/Settings/index.tsx:877
 msgid "Clears all storage data"
-msgstr ""
+msgstr "Borra todos los datos de almacenamiento"
 
 #: src/view/screens/Support.tsx:40
 msgid "click here"
-msgstr "has clic aquí"
+msgstr "clic aquí"
 
 #: src/view/com/modals/DeleteAccount.tsx:208
 msgid "Click here for more information on deactivating your account"
-msgstr ""
+msgstr "Clic aquí para obtener más información sobre la desactivación de su cuenta"
 
 #: src/view/com/modals/DeleteAccount.tsx:216
 msgid "Click here for more information."
-msgstr ""
+msgstr "Clic aquí para obtener más información."
 
 #: src/screens/Feeds/NoFollowingFeed.tsx:46
 #~ msgid "Click here to add one."
-#~ msgstr "Has clic aquí para agregar uno."
+#~ msgstr "Clic aquí para agregar uno."
 
 #: src/components/TagMenu/index.web.tsx:152
 msgid "Click here to open tag menu for {tag}"
-msgstr "Has clic aquí para abrir el menu de {tag}"
+msgstr "Clic aquí para abrir el menu de {tag}"
 
 #: src/components/dialogs/PostInteractionSettingsDialog.tsx:304
 msgid "Click to disable quote posts of this post."
-msgstr ""
+msgstr "Clic para deshabilitar las publicaciones de cotización de esta publicación."
```

Comment:
```suggestion
msgstr "Clic para deshabilitar las citas para esta publicación."
```

"Publicaciones de cotización" makes little sense. "Cotización" usually refers to the price of something. "Citas" is shorter, and used in similar websites.